### DISCUSSION TITLE
Is my formatting in this config right?

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [GriefPrevention/GriefPrevention#1584] [Q&A] [answered]
```
    CommandsRequiringAccessTrust: /sethome, /tpaccept, /pwarp set, /pwarp redefinir, /esethome,
      /pw set, /pw redefinir, /go set, /go redefinir
```

**Comment by Jikoo** [comment, marked as answer]
No. Per the default config, the list is semicolon-separated, not comma-separated. The documentation appears to be very outdated. You're looking for something like `/sethome;/tpaccept;/tpyes` etc.

You also cannot specify arguments to block as far as I'm aware. I believe there's an issue for this open somewhere, though I can't find it with a quick search. You can either block the base command entirely or the blocking will not work.

**Reply by thiagogebrimm** [reply]
A oks, very thanks
